lib/diagnostic.js

```diff
@@ -14,20 +14,20 @@
 /*8*/ };
 
 // Suppose we had declared another function between line 7 and 8. Would that
-// new function have access to the value stored in the variable `laseQuantity`
+// new function have access to the value stored in the variable `laserQuantity`
 // on line 3?
 //
 //     `true` or `false`
 //
-let access = null; // replace null with your response
+let access = true; // replace null with your response
 //
 
-// Which scope(s) may access the variable `laswerQuantity`? Choose one or many
+// Which scope(s) may access the variable `laserQuantity`? Choose one or many
 // of the following:
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-let scopes = [/* your response here */];
+let scopes = ['deathStar'];
```

Comment:
`laserQuantity` is inside the `deathStar` scope, so only things that are also inside the `deathStar` scope can access `laserQuantity`. However, `fire` is defined inside the `deathStar` scope, so its function scope is inside the `deathStar` function scope - consequently, things inside the `fire` scope can access anything inside the `deathStar` scope.

Interestingly, that remains true even if we `return` the `fire` function and pass it out of `deathStar` -- that's the basis of closures.